### ISSUE TITLE
False positive for unused public IP in templates

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule.Rules.Azure#336]
**Description of the issue**

Public IP addresses are common referenced from other resources and won't populate the `Properties.ipConfiguration.id` property.

As a result a false positive is generated by `Azure.PublicIP.IsAttached`.

**Expected behaviour**

Exclude `Azure.PublicIP.IsAttached` when validating from a template.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **0.10.0**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
